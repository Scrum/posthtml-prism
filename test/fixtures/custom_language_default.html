<pre><code class="language-javascript">
  const foo = 'foo'
  console.log(foo)
</code></pre>