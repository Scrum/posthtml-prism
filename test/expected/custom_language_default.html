<pre><code class="language-javascript">
  <span class="token keyword">const</span> foo <span class="token operator">=</span> <span class="token string">'foo'</span>
  console<span class="token punctuation">.</span><span class="token function">log</span><span class="token punctuation">(</span>foo<span class="token punctuation">)</span>
</code></pre>